lib/iris/tests/test_name.py

```diff
@@ -48,6 +51,32 @@ def test_NAMEII_trajectory(self):
             cubes, ("name", "NAMEIII_trajectory.cml"), checksum=False
         )
 
+    def test_NAMEIII__no_time_averaging(self):
+        cubes = iris.load(
+            tests.get_data_path(("NAME", "NAMEIII_no_time_averaging.txt"))
+        )
```

Comment:
Actually, the example file is NAME II format, not NAME III.

The first line of the file that says "NAME III" is in fact describing the version of NAME that *produced* the file, not the version of the output. It can do both versions for compatibility.

I think the main way to tell the difference in output formats - certainly the one iris already uses - is whether it has a "Title" (NAME II) or a "Run name" (NAME III).